### PR TITLE
Adds Some Croptops

### PR DESCRIPTION
**Pull request by Zattarra**
# About The Pull Request

Adds three croptops of varying sleeve lengths. Also adds them to the loadout and clothesmate.

## Why It's Good For The Game

Saw some people asking for this and it seemed simple enough.

## A Port?

Not a port.

## Pre-Merge Checklist

- [x] You tested this on a local server.
- [x] This code did not runtime during testing.
- [x] You documented all of your changes.

## Changelog

:cl:
add: Added the shortcrop, longcrop, and formalcrop.
add: Added croptops to loadout.
add: Added croptops to clothesmate.
imageadd: Gave the croptops sprites.
/:cl:

### CODE REVIEW

**Comment by BongaTheProto** [comment]
can you plz modularize these changes?

**Comment by Zattarra** [comment]
AH SHIT.

**Comment by BongaTheProto** [comment]
the rest of the code and icons can still be modularized

**Comment by Zattarra** [comment]
Yes, I know.

**Comment by Zattarra** [comment]
OH SHIT, THAT'S NOT HOW I DO IT.

**Comment by Zattarra** [comment]
I think that broke more stuff.

**Comment by Zattarra** [comment]
I am absolutely not sure what's up, searched it up on the discord, and maybe there's something with unticking files. No idea what that means.

**Comment by Zattarra** [comment]
Imma try un-modularizing that bit and seeing if it works again.

**Comment by Zattarra** [comment]
I think I made it work with modularization.

**Comment by BongaTheProto** [comment]
seems they don't properly add to the backpack when joining

**Comment by Zattarra** [comment]
Hmm, that is quite strange. I got them to appear in my backpack when added via loadout on my test server.

**Comment by BongaTheProto** [comment]
> Hmm, that is quite strange. I got them to appear in my backpack when added via loadout on my test server.
> 
> It also appears to work when coming in late via shuttle.

Doesn't seem to be the case here
ezgif-4-11d76c8434

**Comment by Zattarra** [comment]
I will look into the error. Perhaps it's a Byond version related issue?

**Comment by Zattarra** [comment]
I am unable to reproduce the bug, I would like to know what version of Byond you are running. It's the only thing I can think of.

**Review by BongaTheProto — CHANGES_REQUESTED** [review]
Well it seems that recently a system to properly handle loadout accessories has been added, so I'd recommend looking into that to fit the croptops better

**Comment by BongaTheProto** [comment]
fix the conflicts pls

File: modular_splurt/code/modules/vending/clothesmate.dm [hunk 1]
```diff
@@ -17,6 +17,9 @@
 		/obj/item/clothing/suit/toggle/tunnelfox = 3,
 		/obj/item/clothing/under/performer = 2,
 		/obj/item/clothing/under/bluedress = 3,
+		/obj/item/clothing/accessory/shortcrop = 3,
+		/obj/item/clothing/accessory/longcrop = 3,
+		/obj/item/clothing/accessory/formalcrop = 3
```
Comment (BongaTheProto) [inline modular_splurt/code/modules/vending/clothesmate.dm:22]:
```suggestion
		/obj/item/clothing/accessory/formalcrop = 3,
```

**Review by BongaTheProto — APPROVED** [review]

**Pull request merged**